gulpfile.js

```diff
@@ -71,33 +74,36 @@ gulp.task('prepare-source', function () {
         }))
         .pipe(rename('freemius-en.po'))
         .pipe(gulp.dest(languagesFolder));
-});
+}
 
 // Push updated po resource to transifex.
-gulp.task('update-transifex', ['prepare-source'], function () {
+function update_transifex() {
+    prepare_source();
     return gulp.src(languagesFolder + 'freemius-en.po')
         .pipe(transifex.pushResource());
-});
+}
 
 // Download latest *.po translations.
-gulp.task('download-translations', ['update-transifex'], function () {
+async function download_translations() {
+    update_transifex();
```

Comment:
@bahiirwa This and all other direct usages of async tasks need refactoring as above.